doc/sphinxext/mne_doc_utils.py

```diff
@@ -262,6 +265,25 @@ def reset_modules(gallery_conf, fname, when):
         mem = sizeof_fmt(process.memory_info().rss)
         print(f"{prefix}{time.time() - t0:6.1f} s : {mem}".ljust(22))
 
+    if fname == "50_configure_mne.py":
```

Comment:
It always bothered me that to build the docs we had to tinker with the user's config. Now we don't.

In principle it might be better someday to make it so that example didn't mess with the real config even when run by a user, but I didn't see an obvious clean way to do it. In the meantime this makes our doc building a bit cleaner at least.